admin/sql/CreateIndexes.sql

```diff
@@ -611,9 +611,15 @@ CREATE INDEX work_tag_raw_idx_tag ON work_tag_raw (tag);
 
 CREATE UNIQUE INDEX work_type_idx_gid ON work_type (gid);
 
--- lowercase indexes for javascript autocomplete
-CREATE INDEX artist_idx_lower_name ON artist (lower(name));
-CREATE INDEX label_idx_lower_name ON label (lower(name));
```

Comment:
What does autocomplete use, just name?

Reply:
The direct search makes use of the indexes in admin/sql/CreateSearchIndexes.sql